tests/bgpd/test_mp_attr.c

```diff
@@ -809,7 +809,26 @@ static struct test_segment {
 		37,
 		SHOULD_ERR,
 	},
-
+	{
+		"IPv4",
+		"IPV4 MP Reach, flowspec, 1 NLRI",
+		{
+			/* AFI / SAFI */ 0x0,
+			AFI_IP,
+			IANA_SAFI_FLOWSPEC,
+			0x00,
+			0x00,
+			0x06,
+			0x01,
+			0x1e,
+			0x1e,
+			0x28,
+			0x28,
+			0x0
+		},
+		(12),
+		SHOULD_PARSE,
```

Comment:
please let's have it readable in modern c : .field = XYZ

Reply:
Can you comment using /* */ each fields of the .data[] for the flowspec content?

Reply:
ok